spkrepo/tests/test_frontend.py

```diff
@@ -24,7 +24,7 @@ def test_get_logged_user(self):
 
 
 class PackagesTestCase(BaseTestCase):
-    def test_get_active_stable(self):
+    def test_get_active(self):
         build = BuildFactory(version__report_url=None, active=True)
```

Comment:
If this is now supposed to be a mirror image of `test_get_not_active`, then perhaps the build factories should be identical by adding or removing this parameter:

```
version__report_url=None
```

Reply:
@hgy59, I'd love to hear your thoughts on these review comments as well.

Reply:
A report url != None means that it is a beta version.
Those tests document that there is not beta label shown anymore on the packages page.

Reply:
Okay. In that case should you also add it to `test_get_not_active(self)`:
https://github.com/SynoCommunity/spkrepo/blob/fed32c9e347c8b706cc16481b7eca3c9f6995d4a/spkrepo/tests/test_frontend.py#L39

Reply:
there is no `test_get_not_active` anymore.

Reply:
Sorry, I was comparing these two:

https://github.com/SynoCommunity/spkrepo/blob/5bce17219319e29dc4b2648b64ea706c28e49a45/spkrepo/tests/test_frontend.py#L30-L31

https://github.com/SynoCommunity/spkrepo/blob/5bce17219319e29dc4b2648b64ea706c28e49a45/spkrepo/tests/test_frontend.py#L43-L44

Reply:
@mreid-tt thanks, I think now I've got it 😃